ServerManager/init.php

```diff
@@ -14,7 +14,7 @@
 require_once 'ServerManager/helpers.php';
 
 // let's get going!
-session_start();
+session_id() or session_start();
```

Comment:
this fixes a notice error when calling "start" if the session is already started